glide-core/src/client/value_conversion.rs

```diff
@@ -287,6 +288,27 @@ pub(crate) fn convert_to_expected_type(
             )
                 .into()),
         },
+        ExpectedReturnType::KeyWithMemberAndScore => match value {
```

Comment:
Add a comment please describing what are you doing here

Reply:
Also, work with Yury to get the Java implementation to match.

Reply:
Right, java client does not support RESP2.